src/main/scala/scalajsGames/games/AstroLander.scala

```diff
@@ -107,30 +107,35 @@ case class AstroLander(bounds: Point, resetGame: () => Unit) extends Game{
   def update(keys: Set[Int]) = {
     lastKeys = keys
     if (fuel > 0){
-      if (keys(37)) craftVel += Point(0.5, 0).rotate(theta + math.Pi / 4)
-      if (keys(39)) craftVel += Point(0.5, 0).rotate(theta - math.Pi / 4)
-      if (keys(40)) craftVel += Point(0.5, 0).rotate(theta)
+      // I changed the first parameter of Point of 0.5 to 0.1, thats makes the ship slowest when you go to left and the right
+      if (keys(37)) craftVel += Point(0.1, 0).rotate(theta + math.Pi / 4)
+      if (keys(39)) craftVel += Point(0.1, 0).rotate(theta - math.Pi / 4)
+      if (keys(40)) craftVel += Point(0.4, 0).rotate(theta)
       fuel -= Seq(keys(37), keys(39), keys(40)).count(x => x)
     }
 
     craftVel += Point(0, 0.2)
     craftPos += craftVel
-
-
-
+    var shipLost = false
     val hit = points.flatMap{ p =>
       val prevIndex = points.lastIndexWhere(_.x < craftPos.x)
+      
+      if (craftPos.x < 3 || craftPos.x > 805 || craftPos.y < -50 ) {
```

Comment:
I think it would be best to: shipLost = craftPos.x =< 1 || craftPos.x >= bounds.x || craftPos.y =< 1 || craftPos.y >= bounds.y

uses a value of 1, because sometimes the ship is lost